.github/scripts/run_consensus.sh

```diff
@@ -17,6 +17,15 @@ MIN_VALIDATOR_COUNT=4
 
 export NODE_IMAGE
 
+if [[ -z $(docker inspect ${NODE_IMAGE} | grep "No such object") ]]; then
```

Comment:
Alternatively, we can just check output status of `docker inspect`, as it is more future-proof, ie output of the command might change etc 
```
if docker inspect "${NODE_IMAGE}" > /dev/null; then
```

Reply:
Like this:
```sh
if [[ $(docker inspect ${NODE_IMAGE} > /dev/null) ]]; then 
    echo "image found"; 
else 
    echo "image not found"; 
fi
```
?
because I'm getting `image not found` even though it is available.